pkg/appconfig/config.go

```diff
@@ -57,32 +57,32 @@ func (e Environment) String() string {
 	}
 }
 
-// Local returns true if the environment is local
+// Local returns true if the environment is local (local development)
 func (e Environment) Local() bool {
 	return e == localEnv
 }
 
-// Test returns true if the environment is local
+// Test returns true if the environment is test (used for local or CI/CD testing)
 func (e Environment) Test() bool {
 	return e == testEnv
 }
 
-// Dev returns true if the environment is local
+// Dev returns true if the environment is dev (dev.easi.cms.gov)
 func (e Environment) Dev() bool {
 	return e == devEnv
 }
 
-// Impl returns true if the environment is local
+// Impl returns true if the environment is Impl (impl.easi.cms.gov)
 func (e Environment) Impl() bool {
 	return e == implEnv
 }
 
-// Prod returns true if the environment is local
+// Prod returns true if the environment is Production (easi.cms.gov)
 func (e Environment) Prod() bool {
 	return e == prodEnv
 }
 
-// Deployed returns true if in a deployed environment
+// Deployed returns true if in a deployed environment (anything in AWS)
 func (e Environment) Deployed() bool {
 	switch e {
 	case devEnv:
```

Comment:
These were just some typos I caught. Not really related to this PR, so I'm gonna pull them out for simplicity's sake